it/src/main/java/com/google/cloud/teleport/it/bigtable/BigtableResourceManagerUtils.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.bigtable;
+
+import static java.lang.Math.min;
+import static org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Preconditions.checkArgument;
+
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Utilities for {@link com.google.cloud.teleport.it.bigtable.BigtableResourceManager}
+ * implementations.
+ */
+public final class BigtableResourceManagerUtils {
+
+  private static final int MIN_CLUSTER_ID_LENGTH = 6;
+  private static final int MAX_CLUSTER_ID_LENGTH = 30;
+
+  private BigtableResourceManagerUtils() {}
+
+  /**
+   * Generates a collection that contains a single BigtableResourceManagerCluster object using the
+   * given parameters.
+   *
+   * @param testId the test id to associate the bigtable cluster to.
+   * @param zone the zone/region that the cluster will be deployed to.
+   * @param numNodes the number of nodes that the cluster will contain.
+   * @param storageType the type of storage to configure the cluster with (SSD or HDD).
+   * @return Collection containing a single BigtableResourceManagerCluster object.
+   */
+  static Iterable<BigtableResourceManagerCluster> generateDefaultClusters(
+      String testId, String zone, int numNodes, StorageType storageType) {
+    checkArgument(
+        testId.length() > MIN_CLUSTER_ID_LENGTH - 3,
```

Comment:
nitpick: Took me a bit to understand why `- 3`. Perhaps extracting `"-c1"` to a constant and use `.length()` can make it cleaner.

Reply:
Fixed.